### ISSUE TITLE
Dynamic slowmode 

### ISSUE THREAD

**Issue by orestis119** [Shadow-Creation-Team/Protector#21]
ousiastika to bot 8a blepei poso grhgora stelnontai mhnumata se kapoio channel kai analogws 8a bazei slowmode h ston author diladi mute (to antispam diladi) h sto channel an stelnoun polla atoma mazi

Creadits: athathin#0691

**Comment by antkarag13** [comment]
ti eipes re trele
(legete credits oxi creadits)

**Comment by 4ngelXgr** [comment]
thanks lol

**Comment by orestis119** [comment]
yes sorry xDDDD

**Comment by Tougrel** [comment]
> 
> 
> Great idea - the thresholds (messages/second?) could be also set by the user 🤔

Yeah we can add this as an option on the dashboard. Whitelist/Blacklist channels will be also an option for this if we are going to add it!

**Comment by Tougrel** [comment]
That's a good idea that we can implement in the moderation system for v0.5 (or a minor update of 0.5 if we don't have time to add it). We will look into this feature once we are done with the dashboard so we can properly design this (as I said in discussion #20 you won't be able to edit the settings via discord).

@z7rwastaken will look into this once he is done with the AntiSpam and other QOL updates for v0.5!
